pkg/apis/v2beta2/config_webhook.go

```diff
@@ -168,6 +169,22 @@ func (r *Config) validateConfig() error {
 			}
 		}
 
+		// Sms AWS provider parameters validation
+		if providers.AWS != nil {
+			if providers.AWS.AccessKeyId != nil {
```

Comment:
The `AccessKeyId` and `SecretAccessKey` can be nil?

Reply:
No, AccessKeyId and SecretAccessKey can't be nil.